narratives/ncov_sit-rep_2020-03-27.md

```diff
@@ -0,0 +1,2734 @@
+* [Updates for Africa](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=15).  
+* [Updates for Oceania](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=17).  
+* [What you can do](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=19).  
+* [Scientific credit](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=20).  
+
+<!-- There is no right side text -->
+
+
+<!-- ##### slide break ##### -->
+<!-- This is left-side text 8-->
+# [Transmission across the U.S. is complex and not well-understood](https://nextstrain.org/ncov/2020-03-27?d=tree,map&f_division=Connecticut,Washington&label=clade:B1&m=div&p=full)
+
+Even within the U.S., transmission patterns are highly complex.
+<br><br>
+For example, recent samples from the Grubaugh lab in Connecticut are both ancestral to and intermingled with samples from Washington state.
+While more data is needed to draw firm conclusions, this suggests that there has been recent transmission between even very distant parts of the country.
+<br><br>
+This highlights that the transmission patterns across the U.S. are far more complex than is represented in the currently available data.
+Additional data will help us understand what is likely a deeply interconnected national epidemic playing out amidst highly heterogeneous responses across states.
+<!-- There is no right side text -->
+
+
+<!-- ##### slide break ##### -->
+<!-- This is left-side text 9-->
+# [Multiple transmission chains circulating in Washington state](https://nextstrain.org/ncov/2020-03-27?d=tree,map&f_country=USA&f_division=Washington&p=full&r=location)
+
+Looking at all the cases in Washington state, we see that they form two clear clusters on the tree.
+<br><br>
+However, the two clusters have different origins: the first cluster is most closely related to samples from China, while the second is most closely related to samples from Europe.
+<br><br>
+This shows that there have been at least two separate introductions leading to independent local transmission chains.
+<!-- There is no right side text -->
+
+
+<!-- ##### slide break ##### -->
+<!-- This is left-side text 10-->
+# [Likely two local transmission chains in California](https://nextstrain.org/ncov/2020-03-27?branchLabel=clade&c=location&f_country=USA&f_division=California&label=clade:A7&p=full&r=location&m=div)
+
+We see similar patterns elsewhere in the U.S.
+<br><br>
+Here, we see several samples from Solano County, California, that are sampled on two different dates but very closely related to one another.
+<br><br>
+While the available data from California is too limited to make firm conclusions, this preliminary evidence is consistent with local transmission within Solano County.
+<br><br>
+Elsewhere in the tree, we see a similar pattern for samples in Santa Clara county.
+<br><br>
+Each county's cluster of cases is likely to due to a separate introduction, based on their relative positions in the tree.
```

Comment:
It's a little unclear what you're referring to here. Each county in the country with cases is where my brain went as a reader. Perhaps clarify that you're only referring to Solano and Santa Clara with something like this?
```suggestion
These counties' clusters of cases are likely to due to separate introductions, based on their relative positions in the tree.
```